sdk/cognitivelanguage/azure-ai-language-text-authoring/samples/async/sample_export_project_async.py

```diff
@@ -0,0 +1,68 @@
+# coding=utf-8
+# ------------------------------------
+# Copyright (c) Microsoft.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+FILE: sample_export_project_async.py
+DESCRIPTION:
+    This sample demonstrates how to export a **Text Authoring** project (async).
+USAGE:
+    python sample_export_project_async.py
+REQUIRED ENV VARS (for AAD / DefaultAzureCredential):
+    AZURE_TEXT_ENDPOINT
+    AZURE_CLIENT_ID
+    AZURE_TENANT_ID
+    AZURE_CLIENT_SECRET
+NOTE:
+    If you want to use AzureKeyCredential instead, set:
+      - AZURE_TEXT_ENDPOINT
+      - AZURE_TEXT_KEY
+OPTIONAL ENV VARS:
+    PROJECT_NAME  # defaults to "<project-name>"
+"""
+
+# [START text_authoring_export_project_async]
+import os
+import asyncio
+from azure.identity import DefaultAzureCredential
+from azure.core.exceptions import HttpResponseError
+from azure.ai.language.text.authoring.aio import TextAuthoringClient
+from azure.ai.language.text.authoring.models import StringIndexType
+
+
+async def sample_export_project_async():
+    # settings
+    endpoint = os.environ["AZURE_TEXT_ENDPOINT"]
+    project_name = os.environ.get("PROJECT_NAME", "<project-name>")
+
+    # create a client with AAD
+    credential = DefaultAzureCredential()
+    async with TextAuthoringClient(endpoint, credential=credential) as client:
+        # project-scoped client
+        project_client = client.get_project_client(project_name)
+
+        # start export (LRO) and handle success/error
+        poller = await project_client.project.begin_export(
+            string_index_type=StringIndexType.UTF16_CODE_UNIT,
+        )
+        try:
+            await poller.result()  # completes with None; raises on failure
+            print("Export completed.")
+            print(f"done: {poller.done()}")
+            print(f"status: {poller.status()}")
+        except HttpResponseError as e:
+            msg = getattr(getattr(e, "error", None), "message", str(e))
```

Comment:
This is too much. You don't need getattr here. [HttpResponseError](https://github.com/Azure/azure-sdk-for-python/blob/5e540a49ba24073cf0803a4597d7e09034a46f82/sdk/core/azure-core/azure/core/exceptions.py#L389-L392) will always have a message. Just print out `e.message`